main.cpp

```diff
@@ -44,17 +47,17 @@ int main()
     input_event ie;
     streamsize sie = static_cast<streamsize>(sizeof(struct input_event));
 
-    while(eventsfile.read((char*)&ie, sie)) {
+    while (eventsfile.read((char *)&ie, sie)) {
```

Comment:
I think you'll want to revert `char *` to `char*`

Reply:
i have nothing against it :)